Procfile

```diff
@@ -0,0 +1,4 @@
+# Run Rails & Webpack concurrently
+# Example file from webpack-rails gem
+rails: bundle exec rails server
```

Comment:
Can we use here https://github.com/simulacre/pry-remote-em or https://github.com/Mon-Ouie/pry-remote? It should decouple pry console from the server.